Pipfile

```diff
@@ -10,7 +10,7 @@ python-dotenv = "==0.9.1"
 "validator.py" = "==1.2.5"
 cryptography = "==2.3"
 bcrypt = "==3.1.4"
-requests = "==2.18.4"
+requests = "==2.19.1"
```

Comment:
https://github.com/MasoniteFramework/core/pull/411 PR has this changes, do not require in this PR.